modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -218,6 +218,17 @@ Any editors using this `highlight_on_focus: true` option, can remove this option
 
 // CCFR here.
 
+=== `sandbox_iframes` editor option is now defaulted to `true`.
+
+In {productname} 6.8.1, `sandbox_iframes` editor option was introduced to support a security vulnerability issue with {productname} loading iframes `unsandboxed` which as a result could allow arbitrary code execution inside {productname} content.
+
+With the release of {productname} 7.0, the default for `sandbox_iframes` editor option has been changed to `true`, constituting a breaking change for users whose editor version is below 7.0.
+
+[IMPORTANT]
+The premium **Enhanced Media Embed** plugin uses `iframes` extensively and are currently no workarounds available except to keep a record of allowed sources. Customers who want to continue to use **Enhanced Media Embed** plugin in {productname} 7.0 will be required to add the option `sandbox_iframes: false` to their {productname} init configuration.
+
+For further details on this option, refer to the xref:security.adoc#sandbox-iframes-option[sandbox_iframes] option, or see xref:migration-from-6x.adoc#sandbox-iframes[migration from 6x guide].
```

Comment:
We should add partials for this option and `sandbox_iframes_exclusions`, so that we can include them in a new `content-filtering.adoc` page, under the `Content` heading in the nav, not just `security.adoc`.

Reply:
will update this once i pull in changes from the other PR so i can verify the build has no failures.